plugins/samples/overwrite_header/BUILD

```diff
@@ -0,0 +1,19 @@
+load("//:plugins.bzl", "proxy_wasm_plugin_cpp", "proxy_wasm_plugin_rust", "proxy_wasm_tests")
+
+licenses(["notice"])  # Apache 2
+
+proxy_wasm_plugin_cpp(
```

Comment:
If possible I'd like to get both C++ and Rust tests for each sample. The Rust plugin will be very similar to this one: https://github.com/GoogleCloudPlatform/service-extensions-samples/blob/main/plugins/samples/add_header/plugin.rs

Reply:
Withdrawn.